src/EA.Weee.Web/Areas/AatfReturn/Views/CheckYourReturn/Index.cshtml

```diff
@@ -113,189 +112,189 @@
                                     <strong>Approval number</strong>
                                 </div>
 
+                                </div>
                             </div>
-                        </div>
-                        @foreach (var pcsdata in aatf.SchemeData)
-                        {
-                            <div class="govuk-grid-row govuk-body govuk-body collapsible-font-size">
-                                <div class="govuk-grid-column-one-half">
-                                    <div class="govuk-grid-column-two-thirds">
-                                        @pcsdata.Scheme.Name
-                                    </div>
+                            @foreach (var pcsdata in aatf.SchemeData)
+                            {
+                                <div class="govuk-grid-row govuk-body govuk-body collapsible-font-size">
```

Comment:
Got a merge conflict here, not sure what the issue is but if you make the merge on your local branch just make sure the changes still apply

Reply:
That was a nasty merge conflict 👎